app/models/order_detail.rb

```diff
@@ -38,7 +38,7 @@ def set_problem_order
   belongs_to :price_policy
   belongs_to :statement, inverse_of: :order_details
   belongs_to :journal
-  belongs_to :order
+  belongs_to :order, inverse_of: :order_details
```

Comment:
So leaving the `inverse_of: :order` on `Order` caused the spec failures, so you're leaving it off for now? Or did you figure something out?